test/ui/ui-get.tokens.test.ts

```diff
@@ -6,7 +6,14 @@ import { Keyring } from "@polkadot/api";
 import BN from "bn.js";
 import { WebDriver } from "selenium-webdriver";
 import { getApi, initApi } from "../../utils/api";
-import { FIVE_MIN, MGA_ASSET_ID } from "../../utils/Constants";
+import {
+  BTC_ASSET_NAME,
```

Comment:
Maybe I would just keep these ASSET1, 2, 3

Reply:
Just to keep it flexible, in case we change the labels

Reply:
I thought about this, but the problem is that we build the locator using the names, 
ie : data-testid="wallet-asset-MGA"
so normally build locators like 
locMGA =  //*[@data-testid='wallet-asset-${tokenName}']